src/App/Types.hs

```diff
@@ -60,3 +62,16 @@ data NinjaGraphCLIOptions = NinjaGraphCLIOptions
   , ninjaScanId :: Text
   , ninjaBuildName :: Text
   }
+
+-- | Feature flags are set in the FOSSA API.
+-- Not all feature flags are relevant to Spectrometer, but those that are can be found here.
+--
+-- Since this is client side, the intent is to use the feature flag state to provide better errors to users,
+-- or for FOSSA to selectively enable experimental or optional features in CLI on an organization's behalf org-wide.
+data FeatureFlag
+  = -- | Enable VSI and Monorepo functionality
+    FeatureFlagVSIMonorepo
```

Comment:
[optional nit] why not name this `FeatureFlagVendoredPackageScanning`? I think having only one identifier between core and here would be helpful